NitroxServer/Communication/Packets/Processors/EntityMetadataUpdateProcessor.cs

```diff
@@ -1,9 +1,11 @@
-﻿using NitroxModel.DataStructures.GameLogic;
+using NitroxModel.DataStructures.GameLogic;
+using NitroxModel.DataStructures.GameLogic.Entities.Metadata;
 using NitroxModel.DataStructures.Util;
 using NitroxModel.Packets;
 using NitroxServer.Communication.Packets.Processors.Abstract;
 using NitroxServer.GameLogic;
 using NitroxServer.GameLogic.Entities;
+using System.Linq;
 
 namespace NitroxServer.Communication.Packets.Processors
```

Comment:
I suggest a file full rewrite, improving what you did so we can already have that "serious implementation" you mentionned below.
First, convert to file-scoped namespace (namespace ``NitroxServer.Communication.Packets.Processors;``)